### PR TITLE
feat(forms): [FormikCurrencyField] Render invalid state without error messsage

### PR DESCRIPTION
**Pull request by diyorbek**
According to the design system form fields should have an invalid state without any validation message. This change should be introduced to all other form components.

The approach I have taken here was already used in `FormikMediaField` (which is not yet moved to this library). The message is not rendered when the field receives `EMPTY_ERROR_MESSAGE` as a validation error message. But the field will have red borders indicating that it is invalid.

### CODE REVIEW

File: packages/forms/src/text/FormikCurrencyField.tsx [hunk 1]
```diff
@@ -46,15 +48,17 @@ export const FormikCurrencyField: ForwardRefExoticComponent<FormikCurrencyFieldP
         name,
         validate,
       });
-      const errorText = touched && error && formatError(error);
+      const hasError = touched && !!error;
+      const errorText =
+        touched && error !== EMPTY_ERROR_MESSAGE && error && formatError(error);
```
Comment (doniyor2109) [inline packages/forms/src/text/FormikCurrencyField.tsx:53]: Could other components need this behavior? If so please add them before hand in this PR

**Review by doniyor2109 — APPROVED** [review]

File: packages/forms/src/text/FormikCurrencyField.tsx [hunk 2]
```diff
@@ -7,6 +7,8 @@ function formatInputError(error: string): ReactNode {
   return error || undefined;
 }
 
+export const EMPTY_ERROR_MESSAGE = 'EMPTY_ERROR_MESSAGE';
```
Comment (timurhamitov) [inline packages/forms/src/text/FormikCurrencyField.tsx:10]: Will we create for every field component its own `EMPTY_ERROR_MESSAGE` constant? Or we will move this one onto a common folder and reuse it?

**Review by timurhamitov — APPROVED** [review]

File: packages/forms/src/text/FormikCurrencyField.tsx (hunk 2 above)
Comment (diyorbek) [reply packages/forms/src/text/FormikCurrencyField.tsx:10]: @timurhamitov I think we'll have one common constant. Let's do it on the next PR.

File: packages/forms/src/text/FormikCurrencyField.tsx (hunk 1 above)
Comment (diyorbek) [reply packages/forms/src/text/FormikCurrencyField.tsx:53]: @doniyor2109 I would need to cover the components with tests. Let's do it on the next PR.

**Pull request merged**